install_config/configuring_nuagesdn.adoc

```diff
@@ -0,0 +1,112 @@
+[[install-config-configuring-nuage-sdn]]
+= Configuring Nuage SDN
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Installation
+
+The VSP integration with {product-title} works for both virtual machines (VMs) and baremetal installations of {product-title}.
+
+Ansible is the preferred method of installing {product-title} with Nuage VSP as the network plugin. Refer to the xref:index.adoc#install-config-index[{product-title} installation and configuration section] for details on how to install {product-title} using Ansible.
+
+=== Installation for single master
+
+In the Ansible nodes file, the following parameters need to be specified in order to setup Nuage VSP as the network plugin.
+
+ # Nuage specific parameters
+ openshift_use_openshift_sdn=False
+ openshift_use_nuage=True
+ os_sdn_network_plugin_name='nuage/vsp-openshift'
+ openshift_node_proxy_mode='userspace'
+ nuage_openshift_monitor_rest_server_port=9443
+
+ # VSP related parameters
+ vsd_api_url=https://192.168.103.200:8443
+ vsp_version=v4_0
+ enterprise=nuage
+ domain=openshift
+ vsc_active_ip=192.168.103.201
+ vsc_standby_ip=192.168.103.202
+ uplink_interface=eth0
+
+ # rpm locations
+ nuage_openshift_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/nuage-openshift-monitor-4.0.X.1830.el7.centos.x86_64.rpm
+ vrs_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/nuage-openvswitch-4.0.X.225.el7.x86_64.rpm
+ plugin_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/vsp-openshift-4.0.X1830.el7.centos.x86_64.rpm
+
+ # Optional parameters
+ nuage_interface_mtu=1460
+ nuage_master_adminusername=admin
+ nuage_master_adminuserpasswd=admin
+ nuage_master_cspadminpasswd=admin
+ nuage_openshift_monitor_log_dir=/var/log/nuage-openshift-monitor
+
+ # Required for brownfield install (where a {product-title} cluster exists without Nuage as the networking plugin)
+ nuage_dockker_bridge=lbr0
+
+=== Installation for multiple masters (HA)
+
+A High Availability (HA) environment can be configured with multiple masters and multiple nodes. There are two supported configuration methods for HA--native HA configuration and configuration using Pacemaker.
```

Comment:
We moved away from using pacemaker for HA deployments in 3.1 (https://access.redhat.com/node/2176281), while it is OK to call out that HA can be achieved, with pacemaker, we should only document how to use an Active Active configuration, to align with the recommendations we make in other parts of the documentation.